setup.py

```diff
@@ -19,7 +19,7 @@
 
 setup(
     name='django-dictiterators',
-    version='1.1.0',
+    version='1.1.1',
```

Comment:
What is the difference between ``'Programming Language :: Python :: 3'`` and ``'Programming Language :: Python :: 3 :: Only'`` ?

Reply:
Yeah, I couldn't find anything about the meaning of ``Only``.

Reply:
I can guess that it could be related to Python 2 vs. 3. "Only Python 3, no support for the old Python 2." It doesn't look like it's important to specify anything with "only", though. Those classifiers are anyway intended for searching packages in the Python Package Index PyPi and they don't affect any real functionality in the package itself. There is even a separate field in setup.py for actually defining the minimum supported Python version: `python_requires`. The classifiers only affect the PyPi search.

None of the following docs really specify how to use classifiers with "only".

* https://pypi.org/classifiers/
* https://pypi.org/search/?c=Programming+Language+%3A%3A+Python+%3A%3A+3+%3A%3A+Only
* https://pypi.org/search/?c=Programming+Language+%3A%3A+Python+%3A%3A+2+%3A%3A+Only
* https://peps.python.org/pep-0301/#distutils-trove-classification
* https://packaging.python.org/en/latest/guides/distributing-packages-using-setuptools/#classifiers
* https://packaging.python.org/en/latest/tutorials/packaging-projects/#configuring-metadata